components/landing/topCollections/TopCollections.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div>
-    <div class="flex justify-between mobile">
-      <div class="title is-2">{{ $t('general.topCollectionsHeading') }}</div>
-      <div class="top-collection-controls buttons items-start pt-2">
+    <div class="flex justify-between mobile pb-9 md:pb-10">
+      <div class="title is-2 md:!mb-0">
+        {{ $t('general.topCollectionsHeading') }}
+      </div>
+      <div class="top-collection-controls buttons items-start pt-2 !mb-0">
```

Comment:
`buttons` is a Bulma class I would use tailwind equivalent 

```css
.buttons {
  align-items: center;
  display: flex;
  flex-wrap: wrap;
  justify-content: flex-start;
}
```

Reply:
done